react-hs-examples/redux/build.sh

```diff
@@ -0,0 +1,6 @@
+stack setup --stack-yaml stackjs.yaml
```

Comment:
I like having `build.sh` within `redux` so could you tell me what I should add to `/scripts/build.sh` specifically, and in what section.